### PR TITLE
[11.0][FIX] sale_order_price_recalculation: set discount from pricelist with discount_policy without_discount

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Yes, very needed with that pricelist policy

**Comment by rafaelbn** [comment]
@sergio-teruel there is nor runbot and travis is :red_circle: could you please check to test? Thanks

File: sale_fixed_discount/models/sale_order.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ class SaleOrderLine(models.Model):
 
     @api.onchange('discount')
     def _onchange_discount(self):
+        super(SaleOrderLine, self)._onchange_discount()
```
Comment (pedrobaeza) [inline sale_fixed_discount/models/sale_order.py]: `res = super(...)`

and return that value at the end (it can contain a domain)

Comment (sergio-teruel) [reply sale_fixed_discount/models/sale_order.py]: Are you sure?

Comment (sergio-teruel) [reply sale_fixed_discount/models/sale_order.py]: done!!..

**Comment by pedrobaeza** [comment]
Let's merge this for fixing the annoying bug

**Pull request merged**